qa/workunits/rados/test_rados_tool.sh

```diff
@@ -534,12 +535,58 @@ function test_append()
   rm -rf ./rados_append_4k ./rados_append_4k_out ./rados_append_10m ./rados_append_10m_out
 }
 
+function test_put()
+{
+  # rados put test:
+  # replicated pool
+  $CEPH_TOOL osd pool create rados_put 100 100 replicated
+
+  # create object
+  dd if=/dev/urandom of=rados_object_1k bs=1KB count=1
+
+  # test put command
+  $RADOS_TOOL -p rados_put put rados_put_obj ./rados_object_1k
+  $RADOS_TOOL -p rados_put get rados_put_obj ./rados_object_1k_out
+  diff ./rados_object_1k ./rados_object_1k_out > /dev/null
+  if [ $? == 0 ]; then
+        echo "Both file are same"
+  else
+        die "Put failed expecting same file content"
+  fi
+
+  # test put command with offset 0
+  $RADOS_TOOL -p rados_put put rados_put_obj_offset_0 ./rados_object_1k --offset 0
+  $RADOS_TOOL -p rados_put get rados_put_obj_offset_0 ./rados_object_offset_0_out
+  diff ./rados_object_1k ./rados_object_offset_0_out > /dev/null
+  if [ $? == 0 ]; then
+        echo "Both file are same"
+  else
+        die "Put with offset=0 failed expecting same file content"
+  fi
+
+  # test put command with offset 100
+  $RADOS_TOOL -p rados_put put rados_put_obj_offset_100 ./rados_object_1k --offset 100
+  $RADOS_TOOL -p rados_put get rados_put_obj_offset_100 ./rados_object_offset_100_out
+  dd if=rados_object_1k seek=100 bs=1c of=rados_object_offset_100_local > /dev/null
```

Comment:
better off using `cmp` which allows us to specify the bytes to skip for FILE1 and FILE2 separately.